ibm/service/vpc/resource_ibm_is_vpc_default_routing_table.go

```diff
@@ -0,0 +1,363 @@
+// Copyright IBM Corp. 2017, 2022 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package vpc
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/validate"
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const ()
+
+func ResourceIBMISVPCDefaultRoutingTable() *schema.Resource {
+	return &schema.Resource{
+		Create:   resourceIBMISVPCDefaultRoutingTableCreate,
+		Read:     resourceIBMISVPCDefaultRoutingTableRead,
+		Update:   resourceIBMISVPCDefaultRoutingTableUpdate,
+		Delete:   resourceIBMISVPCDefaultRoutingTableDelete,
+		Exists:   resourceIBMISVPCDefaultRoutingTableExists,
+		Importer: &schema.ResourceImporter{},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Update: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+		Schema: map[string]*schema.Schema{
+			rtVpcID: {
+				Type:        schema.TypeString,
+				Computed:    true,
+				ForceNew:    true,
+				Description: "The VPC identifier.",
+			},
+			"accept_routes_from_resource_type": {
+				Type:        schema.TypeSet,
+				Optional:    true,
+				Computed:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Set:         schema.HashString,
+				Description: "The filters specifying the resources that may create routes in this routing table, The resource type: vpn_gateway or vpn_server",
+			},
+			rtRouteDirectLinkIngress: {
+				Type:        schema.TypeBool,
+				ForceNew:    false,
+				Default:     false,
+				Optional:    true,
+				Description: "If set to true, this routing table will be used to route traffic that originates from Direct Link to this VPC.",
+			},
+			rtRouteInternetIngress: {
+				Type:        schema.TypeBool,
+				ForceNew:    false,
+				Default:     false,
+				Optional:    true,
+				Description: "If set to true, this routing table will be used to route traffic that originates from the internet. For this to succeed, the VPC must not already have a routing table with this property set to true.",
+			},
+			rtRouteTransitGatewayIngress: {
+				Type:        schema.TypeBool,
+				ForceNew:    false,
+				Default:     false,
+				Optional:    true,
+				Description: "If set to true, this routing table will be used to route traffic that originates from Transit Gateway to this VPC.",
+			},
+			rtRouteVPCZoneIngress: {
+				Type:        schema.TypeBool,
+				ForceNew:    false,
+				Default:     false,
+				Optional:    true,
+				Description: "If set to true, this routing table will be used to route traffic that originates from subnets in other zones in this VPC.",
+			},
```

Comment:
it's computed field hence receiving below error
<img width="1039" alt="Screenshot 2023-02-17 at 10 35 46 AM" src="https://user-images.githubusercontent.com/78336632/219554050-1feb97b2-5b3f-4920-8cd0-b7bb5ea10d62.png">